docs/pages/serving_an_origin.mdx

```diff
@@ -1,6 +1,6 @@
 import ExportedImage from "next-image-export-optimizer";
 
-# Install Pelican Origin
+# Serve a Pelican Origin
 
 The [Pelican](http://pelicanplatform.org/) origin exposes a storage backend like POSIX or S3 to other members of the federation. This will give you the steps to install XRootD as well as how to install Pelican and start up an origin.
```

Comment:
```suggestion
The [Pelican](http://pelicanplatform.org/) origin exposes a storage backend such as  POSIX or S3 to other members of the federation. This will give you the steps to install XRootD as well as how to install Pelican and start up an origin.
```